dockerfiles/alpine_3.16

```diff
@@ -0,0 +1,253 @@
+    LUAROCK_VSHARD_VERSION=0.1.20 \
+    LUAROCK_AVRO_SCHEMA_VERSION=3.0.6 \
+    LUAROCK_EXPERATIOND_VERSION=1.1.1 \
+    LUAROCK_QUEUE_VERSION=1.2.0 \
+    LUAROCK_CONNPOOL_VERSION=1.1.1 \
+    LUAROCK_HTTP_VERSION=1.3.0 \
+    LUAROCK_MEMCACHED_VERSION=1.0.1 \
+    LUAROCK_METRICS_VERSION=0.14.0 \
+    LUAROCK_TARANTOOL_PG_VERSION=2.0.2 \
+    LUAROCK_TARANTOOL_MYSQL_VERSION=2.1.1 \
+    LUAROCK_TARANTOOL_MQTT_VERSION=1.5.1 \
+    LUAROCK_TARANTOOL_GIS_VERSION=1.0.1 \
+    LUAROCK_TARANTOOL_PROMETHEUS_VERSION=1.0.4 \
+    LUAROCK_TARANTOOL_GPERFTOOLS_VERSION=1.0.1
+
+COPY files/gperftools_alpine.diff /
+
+ARG ENABLE_BUNDLED_LIBYAML \
+    LUAJIT_DISABLE_SYSPROF \
+    GC64
+
+RUN set -x \
+    && apk add --no-cache --virtual .run-deps.1 \
+        libstdc++ \
+        readline \
+        openssl \
+        yaml \
+        lz4 \
+        binutils \
+        ncurses \
+        libgomp \
+        lua \
+        tar \
+        zip \
+        zlib \
+        libunwind \
+        icu \
+        ca-certificates \
+    && apk add --no-cache --virtual .build-deps.1 \
+        gcc \
+        g++ \
+        cmake \
+        file \
+        readline-dev \
+        openssl-dev \
+        yaml-dev \
+        bsd-compat-headers \
+        lz4-dev \
+        zlib-dev \
+        binutils-dev \
+        ncurses-dev \
+        lua-dev \
+        musl-dev \
+        make \
+        git \
+        libunwind-dev \
+        autoconf \
+        automake \
+        libtool \
+        linux-headers \
+        go \
+        icu-dev \
+        wget \
+    && : "---------- gperftools ----------" \
+    && mkdir -p /usr/src/gperftools \
+    && git clone "$GPERFTOOLS_REPO" /usr/src/gperftools \
+    && git -C /usr/src/gperftools checkout "$GPERFTOOLS_TAG" \
+    && (cd /usr/src/gperftools; \
+        ./autogen.sh; \
+        ./configure --disable-libunwind --enable-frame-pointers; \
+        make -j ${NPROC}; \
+        cp .libs/libprofiler.so* /usr/local/lib/) \
+    && : "---------- pprof for gperftools ----------" \
+    && : "To avoid of the issue:" \
+    && : "'The master branch is Incompatible with go version < 1.13 #538'" \
+    && : "https://github.com/google/pprof/issues/538" \
+    && : "use latest workable commits with the old GO versions" \
+    && ( export GOPATH=/root/go && \
+       export PATH=${GOPATH}/bin:/usr/local/go/bin:$PATH && \
+       export GOBIN=$GOROOT/bin && \
+       mkdir -p ${GOPATH}/src ${GOPATH}/bin && \
+       go install github.com/google/pprof@latest ) \
+    && : "---------- tarantool ----------" \
+    && mkdir -p /usr/src/tarantool \
+    && git clone "$TARANTOOL_DOWNLOAD_URL" /usr/src/tarantool \
+    && git -C /usr/src/tarantool checkout "$TARANTOOL_VERSION" \
+    && git -C /usr/src/tarantool submodule update --init --recursive \
+    && (cd /usr/src/tarantool; \
+       echo "WARNING: Temporary fix for test/unit/cbus_hang test" ; \
+       git cherry-pick d7fa6d34ab4e0956fe8a80966ba628e0e3f81067 2>/dev/null || \
```

Comment:
Commit "Fix compilation for Alpine 3.5". Is it still relevant? Why do we make a cherry-pick of a commit which is already in our branch?
https://github.com/tarantool/tarantool/commit/d7fa6d34ab4e0956fe8a80966ba628e0e3f81067

Reply:
This file was removed from the patch.